action/action_test.go

```diff
@@ -32,7 +33,7 @@ type mockDriver struct {
 func (d *mockDriver) Handles(imageType string) bool {
 	return d.shouldHandle
 }
-func (d *mockDriver) Run(op *driver.Operation) (driver.OperationResult, error) {
+func (d *mockDriver) Run(context context.Context, op *driver.Operation) (driver.OperationResult, error) {
```

Comment:
To avoid shadowing the package name

```suggestion
func (d *mockDriver) Run(ctx context.Context, op *driver.Operation) (driver.OperationResult, error) {
```